wabt/pom.xml

```diff
@@ -11,9 +11,13 @@
   <packaging>jar</packaging>
 
   <name>Chicory - wabt</name>
-  <description>wabt tools running in pure Java with Chicory</description>
+  <description>wabt tools compiled to pure Java with Chicory</description>
 
   <dependencies>
+    <dependency>
+      <groupId>com.dylibso.chicory</groupId>
+      <artifactId>aot-runtime</artifactId>
```

Comment:
My understanding here is that we want to add to the `bom` only modules that are actually going to make the "supported" `1.0`.
The `aot` is still out of this category imho(needs a bit more investment), I aim for it to be next, and I'm happy to help whenever possible, but I'm a bit on the fence of adding it to the `bom` now as it might create misaligned expectations.

Reply:
With the latest changes in #565, `aot-runtime` is no longer needed at all!